_posts/platform/getting-started/2000-01-01-getting-started-with-elk.md

```diff
@@ -234,5 +234,114 @@ output {
+
+This is where [Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.8/index.html) is needed. This project is designed to let you managed your indices life cycle.
+
+Curator is written in Python. In order to install its dependencies, you can modify the `.buildpacks` file to also contain the Python buildpack.
+The `.buildpacks` file should have the following content:
+
+```
+https://github.com/Scalingo/buildpack-jvm-common.git
+https://github.com/Scalingo/python-buildpack.git
+https://github.com/Scalingo/logstash-buildpack.git
+```
+
+In order to instruct the Python buildpack to install Curator and its dependencies, create a file named `requirements.txt` at the root of your project:
+
+```
+PyYAML==5.3.1
+elasticsearch-curator==5.8.0
+```
+
+Finally you'll need a file to write a script that will run curator every 12h. This can be achieved via a simple bash script.
+Create a file named `curator.sh` with the following content:
+
+```bash
+#!/bin/bash
+set -ex
+
+while true; do
+  echo "Running Curator"
+  curator --config curator.yml log-clean.yml
+  # Run Curator every 12h
+  sleep 43200
+done
+```
+
+Finally you need to tell Scalingo to start Curator.
+This can be achieved by adding the following line to your `Procfile`:
+```yaml
+curator: ./curator.sh
+```
+
+### Configuration
+
+All the hard stuff is now done.
+
+Next step is to configure Curator. First you need to configure how Curator connects to your database:
+
+```
+---
+client:
+  hosts:
+    - ${ELASTICSEARCH_HOST}
+  http_auth: ${ELASTICSEARCH_AUTH}
+logging:
+  loglevel: INFO
+  logfile:
+  logformat: default
+```
+
+Curator cannot use the `ELASTICSEARCH_URL`. You need to define 2 other environment variables on your app duplicating `ELASTICSEARCH_URL` content.
+Hence if your `ELASTICSEARCH_URL` variable is set to `http://user:password@host:port`, you need to define 2 environment variables:
+
+```
+ELASTICSEARCH_HOST=host:port
+ELASTICSEARCH_AUTH=user:password
+```
+
+The last step is to configure your indices life cycle. This is based on your indices names.
+When you configured Logstash, we asked you to configure your index with the following name `sc-apps-%{+YYYY.MM.dd}`.
+We will configure Curator to parse the date contained on those names and delete the ones that are too old.
+
+```yaml
+actions:
+  1:
+    action: delete_indices
+    description: Delete old log indices
+    options:
+      ignore_empty_list: True
+      disable_action: False
+    filters:
+    - filtertype: pattern
+      kind: prefix
+      value: ${LOGS_INDICES_PREFIX}
+    - filtertype: age
+      source: name
+      direction: older
+      timestring: '%Y.%m.%d'
+      unit: days
+      unit_count: ${LOGS_RETENTION_DAYS}
+```
+
+You now need to add two environment variables:
```

Comment:
It's kinda odd to read "two environment variables" but actually only seeing one. I would rephrase to prevent this odd thing. Moreover you don't explain what this environment variable should contain (even though the variable name is almost self-explanatory).

Suggestion:
```suggestion
You now need to add two environment variables. The first environment variable is `LOGS_INDICES_PREFIX`. It configures the index pattern that should be affected by this policy. Setting this variable to `sc-apps` prevent Curator from deleting the other indices that are stored in the same database.

The second environment variable is `LOGS_RETENTION_DAYS`. It configures the retention time of your logs (in days). Setting this variable to `10`, Curator will delete an index if it is 10+ days old.

```
LOGS_INDICES_PREFIX=sc-apps 
LOGS_RETENTION_DAYS=10 
```
```

If this suggestion suits you, you can remove the text below.